src/components/FilterBar.jsx

```diff
@@ -7,9 +7,16 @@ import PropTypes from 'prop-types';
 import ImportExportIcon from '@mui/icons-material/ImportExport';
 
 export default function FilterBar({ openFilter }) {
-  const [{ sortBy }, setQueryParams] = useQueryParams();
+  const [{ sortBy, grades, majors, minAmount, maxAmount }, setQueryParams] =
+    useQueryParams();
   const [anchorEl, setAnchorEl] = useState(null);
 
+  const filterCount =
+    (grades?.length ?? 0) +
+    (majors?.length ?? 0) +
+    (minAmount ? 1 : 0) +
+    (maxAmount ? 1 : 0);
```

Comment:
Observation: I'm noticing we're going to have to remember to update this when we add new filters later on. 🤔

Reply:
yeah maybe adding it to `useQueryParams`and exporting it from there would be a good option.